packages/pcd/webauthn-pcd/package.json

```diff
@@ -10,6 +10,11 @@
     "build": "tsc --noEmit",
     "clean": "rm -rf dist node_modules"
   },
+  "files": [
+    "dist",
+    "./README.md",
+    "./LICENSE"
+  ],
```

Comment:
This is an opportunistic fix, as previously this package would have incorrectly included source files as well as build outputs. I will try to think about ways to lint package.json files, or otherwise ensure that they remain consistent with a template showing the required fields.